### ISSUE TITLE
Found Malware

### ISSUE THREAD

**Issue by darklordsupreme** [bitcookies/winrar-keygen#1]
Just on the Release file executable, I found detection score of 2/74 at VirusTotal. CrowdStrike Falcon and Cynet are claiming that.
Just check and disinfect your executable. Thanks!

**Comment by Sonic-The-Hedgehog-LNK1123** [comment]
Those detections are false positives, if the file contained real malware, way more antivirus engines would have flagged the file as malicious.

Some antivirus programs will flag any keygen or cracking program as "malicious" no matter what, even if the file isn't malicious at all.

If you don't trust a binary executable file, your only option is to compile the code yourself.

**Comment by AKR4PC** [comment]
Yes WinRAR Keygen is flagged as virus in Malwarebytes. Don't  use it. Your privacy will be leaked

**Comment by bitcookies** [comment]
Yes, like @Sonic-The-Hedgehog-LNK1123 said, If you don't trust a binary executable file, your only option is to review the source code yourself and compile it yourself.

Or, you can try running it in an isolated environment. 🐞

The information I can provide is that WinRAR Keygen does not detect the threat in Kaspersky.

**Comment by Hazuki-san** [comment]
Wrong, this file is completely safe. These are just false positives.

**Comment by bitcookies** [comment]
The update supports **Github Actions**. 🎉

Now you can run it in the cloud without worrying about security issues, please refer to the [README](https://github.com/bitcookies/winrar-keygen/blob/master/README.md) for details.
